localmodel-agent.Dockerfile

```diff
@@ -26,4 +27,6 @@ RUN /opt/app-root/src/go/bin/go-licenses save --save_path third_party/library ./
 FROM gcr.io/distroless/static:nonroot
 COPY --from=builder /go/src/github.com/kserve/kserve/third_party /third_party
 COPY --from=builder /go/src/github.com/kserve/kserve/localmodelnode-agent /manager
+USER 1000:1000
```

Comment:
should we use ubi for final image?

Reply:
Indeed, we need to update it for ODH.

Reply:
Yeah we should. Do you mind if it's done in a follow up PR?